docs/domain/03_aggregates.md

```diff
@@ -0,0 +1,67 @@
+# Aggregates
+
+Aggregates are part of DDD. To create one you have to create entity class inheriting [`IAggregateRoot`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs) or [`IAggregateRootWithoutOptimisticConcurrency`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs).
+
+## [`IAggregateRoot`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs)
+
+In order to create aggregate root you should create class inheriting it. [`IAggregateRoot`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs) requires you to specify Id type.
```

Comment:
```suggestion
[`IAggregateRoot`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs) requires you to specify Id type.
```

The first sentence is slightly redundant - it is basically a rephrasing of the first paragraph.

Reply:
I'm also not really sure that talking about the `Id` type here adds anything - let's talk about the `Id` itself (why we need that).